airflow/utils/dag_processing.py

```diff
@@ -121,6 +143,45 @@ def get_task_special_arg(self, task_id, special_arg_name):
             return None
 
 
+class SimpleTaskInstance(object):
```

Comment:
it seems  the class doesn't have method. how about using Namedtuple instead of creating a class which will look a lot simpler?

Reply:
Good to know about Namedtuple :D Tho I plan to add some more stuff into this class including methods([PR](https://github.com/yrqls21/incubator-airflow/pull/4) WIP). I might explore to see if there's cleaner way to write it as the class indeed is going to be mostly holding variables.